nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/resources/docs/org.apache.nifi.processors.gcp.vision.StartGcpVisionAnnotateFilesOperation/additionalDetails.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
```

Comment:
There are multiple suggestions I'd like to make, here's how the document would look like based on those:
```html
<!DOCTYPE html>
<html lang="en" xmlns="http://www.w3.org/1999/html">
<!--
      Licensed to the Apache Software Foundation (ASF) under one or more
      contributor license agreements.  See the NOTICE file distributed with
      this work for additional information regarding copyright ownership.
      The ASF licenses this file to You under the Apache License, Version 2.0
      (the "License"); you may not use this file except in compliance with
      the License.  You may obtain a copy of the License at
          http://www.apache.org/licenses/LICENSE-2.0
      Unless required by applicable law or agreed to in writing, software
      distributed under the License is distributed on an "AS IS" BASIS,
      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
      See the License for the specific language governing permissions and
      limitations under the License.
    -->

<head>
    <meta charset="utf-8"/>
    <title>Amazon Textract</title>
    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css"/>
</head>
<body>

<h1>Google Cloud Vision - Start Annotate Files Operation</h1>
<p>
    Prerequisites
<ul>
    <li>Make sure Vision API is enabled and the account you are using has the right to use it</li>
    <li>Make sure the input file(s) are available in a GCS bucket</li>
</ul>
</p>
<h3>Usage</h3>
<p>
    StartGcpVisionAnnotateFilesOperation is designed to trigger file annotation operations. This processor should be used in pair with the GetGcpVisionAnnotateFilesOperationStatus Processor.
    Outgoing FlowFiles contain the raw response to the request returned by the Vision server. The response is in JSON format and contains the result and additional metadata as written in the Google Vision API Reference documents.
</p>

<h3>Payload</h3>
<p>
    The JSON Payload is a request in JSON format as documented in the <a href="https://cloud.google.com/vision/docs/reference/rest/v1/files/asyncBatchAnnotate" target="_blank">Google Vision REST API reference document</a>.
    Payload can be fed to the processor via the <code>JSON Payload</code> property or as a FlowFile content. The property has higher precedence over FlowFile content.
    Please make sure to delete the default value of the property if you want to use FlowFile content payload.
    A JSON payload template example:
</p>
<code>
    <pre>
{
    "requests": [
        {
            "inputConfig": {
                "gcsSource": {
                    "uri": "gs://${gcs.bucket}/${filename}"
                },
                "mimeType": "application/pdf"
            },
            "features": [{
                    "type": "DOCUMENT_TEXT_DETECTION",
                    "maxResults": 4
                }],
            "outputConfig": {
                "gcsDestination": {
                    "uri": "gs://${gcs.bucket}/${filename}/"
                },
                "batchSize": 2
            }
        }]
}
    </pre>
</code>
<h3>Features types</h3>
<ul>
    <li>TEXT_DETECTION: Optical character recognition (OCR) for an image; text recognition and conversion to machine-coded text. Identifies and extracts UTF-8 text in an image.</li>
    <li>DOCUMENT_TEXT_DETECTION: Optical character recognition (OCR) for a file (PDF/TIFF) or dense text image; dense text recognition and conversion to machine-coded text.</li>
</ul>
You can find more details at <a href="https://cloud.google.com/vision/docs/features-list" target="_blank">Google Vision Feature List</a>

<h3>Example: How to setup a simple Annotate Image Flow</h3>
<p>
    Prerequisites
</p>
<p>
    <ul>
    <li>Input files should be available in a GCS bucket</li>
    <li>This bucket must not contain anything else but the input files</li>
    </ul>
</p>
<p>Create the following flow</p>
<img src="vision-annotate-files.png" style="height: 50%; width: 50%"/>
<p>
Keep the default value of JSON PAYLOAD property in StartGcpVisionAnnotateImagesOperation
</p>
<p>
Execution steps:
    <ul>
        <li>ListGCSBucket processor will return a list of files in the bucket at the first run.</li>
        <li>ListGCSBucket will return only new items at subsequent runs.</li>
        <li>StartGcpVisionAnnotateFilesOperation processor will trigger GCP Vision file annotation jobs based on the JSON payload.</li>
        <li>StartGcpVisionAnnotateFilesOperation processor will populate the <code>operationKey</code> flow file attribute.</li>
        <li>GetGcpVisionAnnotateFilesOperationStatus processor will periodically query status of the job.</li>
    </ul>
</p>
</body>
</html>
```

Reply:
I've added you suggestions to the PR. thank you for the remarks.